bin/stress-test/src/seeding/metrics.rs

```diff
@@ -163,8 +163,16 @@ impl Display for SeedingMetrics {
 
 /// Gets the size of the store file and its WAL file.
 fn get_store_size(dump_file: &Path) -> u64 {
-    let store_file_size = std::fs::metadata(dump_file).unwrap().len();
-    let wal_file_size =
-        std::fs::metadata(format!("{}-wal", dump_file.to_str().unwrap())).unwrap().len();
+    let store_file_size = fs_err::metadata(dump_file)
+        .unwrap_or_else(|_| panic!("Dumpfile always exists: {}", dump_file.display()))
```

Comment:
I thought `fs_err` was meant to handle the filepath errors?

So something like `.expect("dumpfile should exist")` would automatically add in the expected filepath as part of the panic dump?